packages/app/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@backstage/app",
+  "name": "example-app",
```

Comment:
but you didn't rename the actual directory?

Reply:
Yeah no, wondering if we should

Reply:
mm, i am not entirely sure but thought that maybe that communicates the purpose even more strongly than the package name ... the directory is the first thing you see

Reply:
can ship this though and discuss that in a separate PR?

Reply:
sure.

Reply:
approve? :grin: